prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -148,8 +182,20 @@ class FhirPathCustomResolver : FHIRPathEngine.IEvaluationContext {
         focus: MutableList<Base>?,
         functionName: String?,
         parameters: MutableList<MutableList<Base>>?
-    ): MutableList<Base> {
-        throw NotImplementedError("Not implemented")
+    ): MutableList<Base>? {
+        check(focus != null)
+        check(!functionName.isNullOrBlank())
+        return (
+            when (functionName) {
+                "RSAddressUse" -> {
+                    CustomFHIRCommands.convertAddressUse(focus)
+                }
+                "RSPhoneNumberCountryCode" -> {
+                    CustomFHIRCommands.getPhoneCountryCode(focus)
+                }
+                else -> mutableListOf<Base>()
```

Comment:
I am wondering, can you test with a function name that does not exist?  I am thinking this may result in an empty value instead of an error (I think it is supposed to be some exception, but not sure).

Reply:
After turning it into an enum, it should throw an error now!